### PR TITLE
Error. Delete pr please.

### PR DESCRIPTION
**Pull request by SylvainNeau**
A propos des Fel Stalker qui ne doivent pas pour être controlés par les démonistes.

### CODE REVIEW

File: database [hunk 1]
```diff
@@ -0,0 +1 @@
+Subproject commit 9ed1e656eb007866293d2c67b296fbccd82d4b85
```
Comment (billy1arm) [inline database]: @SylvainNeau - Could you remove this line ?

Could you also pr against Rel20, not master ?

Thank you :)

Comment (SylvainNeau) [inline database]: Yes, asap.
I'm new at github, and still learning how to use it.  :(

Comment (SylvainNeau) [inline database]: I think it's good now.

**Comment by SylvainNeau** [comment]
I've pr on rel20. You could delete this pr.
sorry.

**Pull request closed without merging**